sdk/resourcemanager/Azure.ResourceManager/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0004: DO provide both asynchronous and synchronous variants for all service methods.", Justification = "")]
```

Comment:
This is another library where we shouldn't be opting out of existing warnings.  Opting out of new warnings might make sense in some cases, but opting out of existing warnings is introducing technical debt, which we shouldn't do without a very strong justification.

Reply:
And can we avoid ever creating global suppressions even when we do need to suppress warnings? Without optional context, they are global to an entire repo and I would hope our warning contexts can be specific enough we can opt out specific warnings - not all past, present, and _future_ warnings. That sets a dangerous precedent.

Reply:
It's removed.